webpack.config.js

```diff
@@ -87,7 +87,7 @@ module.exports = {
       inject: false
     }),
     new ExtractTextWebpackPlugin({
-      filename: 'assets/consort-report.[contenthash].css'
+      filename: 'assets/consort-report.[hash].css'
```

Comment:
It sounds like webpack introduced a `contenthash` variable. https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/763

Reply:
Yeah, we should be using mini-css-extract-plugin with Webpack 4. You may have run into this by copying from Vizr, which is still Webpack 3.

Reply:
That's where it came from. I created REDDEV-597 to handle the change.